guillotina_ratelimit/__init__.py

```diff
@@ -0,0 +1,62 @@
+from guillotina import configure
+
+
+app_settings = {
+    "ratelimit": {
+        "global": None,
+        "state_manager": "redis",
+        "redis_prefix_key": 'ratelimit-'
+    }
+}
+
+_registered_service_ratelimits = {}
+
+
+def register_ratelimits(klass, config):
+    if klass in _registered_service_ratelimits:
+        # do not register twice
+        raise Exception('Rate-limit was configured twice!')
+    _registered_service_ratelimits[klass] = config
+
+
+def get_service_ratelimits(method, view_name):
```

Comment:
Not sure if this is optimal, but that was the only way I could come up with without subclassing the `@configure.service` from guillotina.

Do you guys think that would be better?